haskell/skynet.cabal

```diff
@@ -0,0 +1,28 @@
+name:                skynet
+version:             0.1.0.0
+synopsis:            Simple project template from stack
+description:         Please see README.md
+homepage:            http://github.com/bitemyapp/skynet#readme
```

Comment:
It's from the stack template, an oversight. I can remove it if it's an issue. I didn't expect it would get uploaded to Hackage so shouldn't have any relevance.